Responsive design for mobile devices (mainPage) and responsive design for mobile devices and tab devices (gptArena ) .

anushkaDev9 opened the pull request:
Responsive design for mobile devices (mainPage) and responsive design  for  mobile devices and tab devices (gptArena ) .
![image](https://github.com/klnamv/gpt_arena/assets/83566191/e6268ae8-6822-455c-960c-b77ee16511ab)
![image](https://github.com/klnamv/gpt_arena/assets/83566191/ae3f952e-63ee-4e18-84ff-bb899c471487)

anushkaDev9 committed:
responsive Design

klnamv commented:
@anushkaDev9 
I absolutely love what you've done, but you need to fix a few things:

**mainPage:**
- reduce the text (Welcome to the GPT arena);
- fix the image size(height).
<img width="240" alt="image" src="https://github.com/klnamv/gpt_arena/assets/117654777/a1e557ad-d8da-4422-a758-83cda5cd1b40">



**gptArena:**
- container size(height);
- size of the submit button; 
- reduce the size of the text (GPT Arena);
<img width="240" alt="image" src="https://github.com/klnamv/gpt_arena/assets/117654777/4554c63d-b260-41a1-863d-b3f2ad8df3bd">



**!! Important !!**
The GPT-Arena website design presents all content in one window, avoiding the need for scrolling.

anushkaDev9 committed:
Required Changes

Updated
reduce the text (Welcome to the GPT arena);
fix the image size(height).
container size(height);
size of the submit button;
reduce the size of the text (GPT Arena);

anushkaDev9 commented:
I have added the required changes please let me know if you can see it and any further changes let me know .Thank you

klnamv commented:
Wow, looks like a design, I really like your work.

And the last thing you need to change:
- align the text and elements on the mainPage to the left edge (Welcome to the GPT arena and button)

anushkaDev9 committed:
Required changes (2)

updated
align the text and elements on the mainPage to the left edge (Welcome to the GPT arena and button)

anushkaDev9 commented:
Thank you.
I have again changed and updated .Let me know if this is good?

anushkaDev9 committed:
Responsive

Responsive devices for  iphone se(375px)
pixel 7 (412px)
samsung galaxy (412px)
ipad mini
